xml/deployment_firstboot.xml

```diff
@@ -0,0 +1,870 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//Novell//DTD NovDoc XML V1.0//EN" "novdocx.dtd"
+[
+  <!ENTITY % NOVDOC.DEACTIVATE.IDREF "INCLUDE">
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter id="cha.deployment_firstboot">
+ <title>Deploying Customized Preinstallations</title>
+ <para>
+  Rolling out customized preinstallations of &productname; to a large number
+  of identical machines spares you from installing each one of them
+  separately and provides a standardized installation for the end users.
+  With &yast; Firstboot, create customized preinstallation images and
+  determine the workflow for the final personalization steps that involve
+  end user interaction (as opposed to &ay;, which allows completely
+  automated installations; for more information, see
+  <xref linkend="cha.deployment.autoinst"/>).
+ </para>
+ <para>
+  Creating a custom installation, rolling it out to your hardware, and
+  personalizing the final product involves the following steps:
+ </para>
+ <procedure>
+  <step>
+   <para>
+    Prepare the master machine whose disk needs to be cloned to the client
+    machines. For more information, refer to
+    <xref linkend="sec.fb.master"/>.
+   </para>
+  </step>
+  <step>
+   <para>
+    Customize the firstboot workflow. For more information, refer to
```

Comment:
```suggestion
    Customize the Firstboot workflow. For more information, refer to
```

Reply:
Leaving as-is. I searched the docs, and 'firstboot'  is mostly lower-case. Maybe it should be capitalized, but I don't want to deal with it right now.